packages/react-form/src/start/utils.ts

```diff
@@ -1,3 +1,15 @@
-import { createCookie } from '@remix-run/node'
+import { getCookie, setCookie } from '@tanstack/react-start/server'
+import * as devalue from 'devalue'
 
-export const _tanstackInternalsCookie = createCookie('_tanstack_form_internals')
+export const _tanstackInternalsCookie = {
+  name: 'tanstack-internals',
```

Comment:
Let's keep `_tanstack_form_internals` as name

Reply:
done on f443ca3a04d2 !

Reply:
`_tanstack_form_internals` with `form` in it please, just in case other tanstack libraries want to use a similar name

Reply:
ok I gonna change it!

Reply:
done with bda493dd0da7